hfc/fabric/client.py

```diff
@@ -444,13 +444,12 @@ def get_channel(self, name):
 
         return self._channels.get(name, None)
 
-    # TODO pass enveloppe directly
-    # TODO channel_create and channel_update are almost the same, refacto
-    async def channel_create(self, orderer, channel_name, requestor,
-                             config_yaml=None, channel_profile=None,
-                             config_tx=None):
+    # TODO pass envelope directly
+    def get_headers_create_or_update_channel(self, orderer, channel_name, requestor,
+                                             config_yaml=None, channel_profile=None,
+                                             config_tx=None, signatures=None):
         """
-        Create a channel, send request to orderer, and check the response
+        Get headers to send request to orderer, and check the response
```

Comment:
This method will prepare the headers that are used for creating or updating channel.
Should make clear in the doc string.